babel/src/test/resources/sql/postgresql.iq

```diff
@@ -57,3 +57,8 @@ SELECT * FROM sal_emp;
 NAME, PAY_BY_QUARTER, SCHEDULE
 Bill, [10000, 10000, 10000, 10000], [[meeting, lunch], [training, presentation]]
 !ok
+
+select to_char(timestamp '2022-06-03 12:15:48.678', 'YYYY-MM-DD HH24:MI:SS.MS TZ');
```

Comment:
Can you add more tests for `toChar(DataContext ctx, long timestamp, String pattern)` function in org.apache.calcite.test.SqlFunctionsTest ?